### ISSUE TITLE
Promote ADR-022 to accepted, and settle where ADR status lives

### ISSUE THREAD

**Issue by tosin2013** [tosin2013/mcp-adr-analysis-server#1493]
ADR-022's thinking is finished. Its `## Decision Outcome` (line 54) is written, `### Why not the others` (line 63), `## Consequences` (77) and `## Confirmation` (103) are all filled in. Only the frontmatter is unpromoted:

```yaml
status: proposed
```

Flip it to `accepted`. That is the whole first half, and it unblocks #1466.

## Second half: settle where ADR status lives

Adopting MADR is the decision that owns this, so settle it here rather than leaving a third dialect to accumulate. The corpus currently mixes:

```
ADR-001..020   ## Status heading, value on a following line
ADR-021        ## Status heading
ADR-022, 023   YAML frontmatter, lowercase MADR vocabulary
```

`scripts/check-adr-drift.sh` already parses four dialects — heading (57%), bullet (19%), inline (16%), YAML (2%), frequencies measured by repo-governor's adapter across 439 real ADRs — and normalises MADR's lowercase to Title Case so `accepted` vs `Accepted` does not report as false drift. So the checker copes today.

Coping is not the same as deciding. Pick one location, say so in ADR-022 (which is where the format decision belongs), and record whether the existing 21 ADRs get migrated or grandfathered. Grandfathering is a legitimate answer — say it explicitly rather than leaving it implied.

## Acceptance

- [ ] `status: accepted` in ADR-022's frontmatter
- [ ] ADR-022 states where status lives for new ADRs
- [ ] Migrate-or-grandfather recorded explicitly for the existing corpus
- [ ] `bash scripts/check-adr-drift.sh` at or below baseline
- [ ] #1466's stated blocker updated

Fully parallel — nothing else in this milestone depends on it.
